distribution/docs/src/main/resources/_contents/_configuring/configuring-from-import-contents.adoc

```diff
@@ -0,0 +1,78 @@
+
+The Configuration Export/Import capability allows administrators to export the current ${branding} configuration and use it as a starting point for a new installation.
+This is useful when upgrading or expanding use of ${branding} where an identical configuration of multiple instances is desired.
+
+[IMPORTANT]
+====
+* Importing configuration files is only guaranteed to work when importing files from the same ${branding} version.
+  Importing from a different version is not recommended as it may cause the new ${branding} instance to be incorrectly configured and become unusable.
+* All configurations editable in the Admin Console will be exported to `<${branding}_HOME>/etc/exported/etc`.
+* All other configuration files (system configurations) will be put under `etc/exported/<ID>` followed by their relative path from `${branding}_HOME`.
+  For instance, if a key store file was located under `<${branding}_HOME>/keystores/keystore.jks`, it will be exported to `<${branding}_HOME>/etc/exported/<ID>/keystore/keystore.jks`.
+* To keep the export/import process simple and consistent, all system configuration files are required to be under the `${branding}_HOME` directory.
+====
+
+==== Exporting Existing Configurations from ${admin-console}
+
+You can export the current system configurations using the ${admin-console}.
+This is useful for migrating from one running instance to another.
+
+To export the current system configurations, follow these instructions:
+
+. Select the `System` tab (next to the Applications tab) image:exporting_configuration_step1.png[Exporting Step 1,${image.width}]
+. Click the `Export Configuration` button image:exporting_configuration_step2.png[Exporting Step 2,${image.width}]
+. Fill out the form, specifying the _destination_ for the export.  A relative path will be relative to `<${branding}_HOME>`. image:exporting_configuration_step3.png[Exporting Step 3,${image.width}]
+. Click the `Start Export` button
+. If there are no warnings or errors, the form will automatically close upon finishing the export
+
+==== Export Existing Configuration Settings from ${command-console}
+
+To export the current ${branding} configuration:
+
+. Using the ${command-console}, type in `migration:export <directory>`. This command creates the exported configuration files that are saved to the specified directory. If no directory is specified it will default to `<${branding}_HOME>/etc/exported`
+. Zip up the exported files in the export directory.
+----
+cd  <${branding}_HOME>/etc/exported
+zip exportedFiles.zip *
+----
+
+==== Troubleshooting Common Warnings or Failures
+
+If export is unsuccessful, use this list to verify the correct configuration.
+
+===== Insufficient Write Permissions
+
+In the following case, the directory the user tried to export to had permissions set to read only.
```

Comment:
Should we use something else than `In the following case` to begin those paragraph?